reposerver/repository/repository.go

```diff
@@ -766,7 +615,7 @@ func helmTemplate(appPath string, repoRoot string, env *v1alpha1.Env, q *apiclie
 				return nil, err
 			}
 			defer file.Close()
-			templateOpts.Values = append(templateOpts.Values, p)
+			templateOpts.Values = append(templateOpts.Values, helm.ResolvedFilePath(p))
```

Comment:
Won't this always be out-of-path? 

> If dir is the empty string, CreateTemp uses the default directory for temporary files

https://pkg.go.dev/os#CreateTemp

We could write the values file into the repo directory temporarily. If we're allowing concurrent use of the repo, there's a chance someone could snatch someone else's values file, so we should probabluy upgrade from `ioutil.TempFile` to something containing UUIDv4.

Reply:
Agree about using UUIDv4 - updated.

We want to make sure attacker won't access out-of-path file. In this case, path is generated with no user input so safe to use out-of-path location.